test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -767,6 +766,60 @@ internal static partial class InlineArrayIndexerExtensions
         this.AssertGeneratedType("MainAVIHeader", expected, expectedIndexer);
     }
 
+    [Theory, MemberData(nameof(TFMData))]
+    public void FixedLengthInlineCharArraysHaveToStringWithLength(string targetFramework)
+    {
+        this.compilation = this.starterCompilations[targetFramework];
+
+        const string expected = @"
+                /// <summary>
+                /// Copies the fixed array to a new string up to the specified length regardless of whether there are null terminating characters.
+                /// </summary>
+                /// <exception cref = ""ArgumentOutOfRangeException"">
+                /// Thrown when <paramref name = ""length""/> is less than <c>0</c> or greater than <see cref = ""Length""/>.
+                /// </exception>
+                internal string ToString(int length)
+                {
+                    if (length < 0 || length > Length)
+                        throw new ArgumentOutOfRangeException(nameof(length), length, ""Length must be between 0 and the fixed array length."");
+                    unsafe
+                    {
+                        fixed (char *p0 = &_0)
+                            return new string (p0, 0, length);
+                    }
+                }";
```

Comment:
With #303, whitespace is becoming much more malleable. The few verbatim tests we have already were the very last that I got working. I don't want more of them, as we're likely to make additional whitespace changes in the future and I don't want the tax of matching this expectation or updating the tests. Please just assert the member exists, and if you want to test functionality, do that in the `BasicTests.cs` file.

Reply:
Okay. Testing the functionality is super important if the primary output (the source itself) is no longer tested, so I'll probably take a while on this. You didn't ask for input, so I won't belabor the point further.